packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -81,8 +82,11 @@ import {
   afterActiveInstanceBlur,
   getCurrentEventPriority,
   supportsMicrotasks,
+  supportsResources,
   errorHydratingContainer,
   scheduleMicrotask,
+  prepareToRender,
+  cleanupAfterRender,
```

Comment:
Maybe rename these imports in this context because they're a bit too generic in this context.

Reply:
lmk what you think with the latest change